lib/tpaexec/architecture.py

```diff
@@ -1124,7 +1124,7 @@ def update_repos(self, cluster_vars):
         # BDR.update_cluster_vars), and we don't need to do anything more (and
         # we must not configure any edb_repositories).
 
-        if "tpa_2q_repositories" in cluster_vars:
+        if "tpa_2q_repositories" in cluster_vars and cluster_vars["tpa_2q_repositories"]:
```

Comment:
Would it be more readable?

`if cluster_vars.get("tpa_2q_repositories", None):`